### PR TITLE
add amazonlinux to travis matrix

### PR DESCRIPTION
**Pull request by lamont-granquist**

### CODE REVIEW

**Comment by thommay** [comment]
i like amazonlinus as a name.

**Comment by lamont-granquist** [comment]
lulz amazon linux and epel.

i can't wait to rip amazon linux out of the rhel family.

**Comment by lamont-granquist** [comment]
@chef/client-core here you go, amazon linux testing on every PR...

File: kitchen-tests/cookbooks/base/recipes/default.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
   gpgkey "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-#{node['platform_version'].to_i}"
   gpgcheck true
   mirrorlist "https://mirrors.fedoraproject.org/metalink?repo=epel-#{node['platform_version'].to_i}&arch=$basearch"
-  only_if { node["platform_family"] == "rhel" }
+  only_if { node["platform_family"] == "rhel" && node["platform"] != "amazon" }
```
Comment (thommay) [inline kitchen-tests/cookbooks/base/recipes/default.rb]: saaaaad panda.

**Pull request merged**